src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -46,6 +46,7 @@ import 'sql/workbench/contrib/welcome/page/browser/az_data_welcome_page'; // {{S
 import { IHostService } from 'vs/workbench/services/host/browser/host';
 import { IProductService } from 'vs/platform/product/common/productService';
 
+const fs = require('fs');
```

Comment:
we can't use an `import` for this.  I also don't think this will work in web-mode, IIRC.